### PR TITLE
Add dockerfile build in Makefile

### PR DESCRIPTION
**Pull request by sufuf3**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,3 +65,7 @@ src.test-coverage:
 check-govendor:
 	$(info check govendor)
 	@[ "`which $(GO_VENDOR)`" != "" ] || (echo "$(GO_VENDOR) is missing"; false)
+
+## dockerfiles/ ########################################
+docker.build:
+	docker build -t="sdnvortex/cv-server:latest" ./dockerfiles/
```
Comment (hwchiu) [inline Makefile]: use the `dockerfiles/dockerfile` as the dependency of target `docker.build`
Please rebase master and check the build won't fail.

By the way, use the variable for tag, such as
```
--tag $(DOCKER_REGISTRY)/$(PROJECT):$(IMAGE_TAG) \
--file $< .
```

Comment (hwchiu) [reply Makefile]: Remove `ARG BRAND=aurora`

File: Makefile [hunk 2]
```diff
@@ -6,6 +6,10 @@ GO           = go
 GO_VENDOR    = govendor
 MKDIR_P      = mkdir -p
 
+## For docker
+DOCKER_REGISTRY = sdnvortex
+IMAGE_TAG = latest
+PROJECT = cv-server
```
Comment (hwchiu) [inline Makefile]: project is vortex.

File: dockerfiles/Dockerfile [hunk 3]
```diff
@@ -2,20 +2,20 @@
...
-COPY Makefile /go/src/bitbucket.org/linkernetworks/vortex
+COPY . /go/src/github.com/linkernetworks/vortex/src
+COPY Makefile /go/src/github.com/linkernetworks/vortex
 
 ENV CGO_ENABLED 0
 ENV GOOS linux
 ENV TIMEZONE "Asia/Shanghai"
 ARG BUILD_MODE=release
-ARG BRAND=aurora
+ARG BRAND=vortex
```
Comment (yarencheng) [inline dockerfiles/Dockerfile]: @hwchiu BRAND should be removed not changed?

Comment (sufuf3) [reply dockerfiles/Dockerfile]: This BRAND is for L24.

Comment (hwchiu) [reply dockerfiles/Dockerfile]: We don't need the BRAND here, that's for our previous project.

Comment (hwchiu) [reply dockerfiles/Dockerfile]: Also modify the L24 to remove the BRAND, Thanks

**Comment by sufuf3** [comment]
I fixed it.
@hwchiu would you please help me review it?
Thanks.

**Pull request merged**